README.md

```diff
@@ -1,2 +1,105 @@
-# developer-guides
-Guides and documentation for web developers at the City of Austin
+# Developer Guides
+
+[http://developer-guides.austintexas.io](http://developer-guides.austintexas.io)
+
+---
+
+:construction: This is a work in progress
+
+This repo contains extensive documentation around workflows, best practices, tools, and architectural considerations for web developers at the City of Austin.
+
+The site uses the [Zilker Jekyll theme](https://github.com/cityofaustin/zilker-theme) for UI and navigation handling. Future changes might be required to frontmatter and `_config.yml` based on future iterations. Zilker is currently undergoing structural changes to nav, layouts, and CSS.
+
+Learn more about Jekyll at [the Jekyll documentation site](https://jekyllrb.com/docs/home/)
+
+## Quick Start
+
+1. Clone the repo  
+
+   ```
+   $ git clone https://github.com/cityofaustin/developer-guides
+   ```
+
+2. `cd` into the project directory and install gems  
```

Comment:
@briaguya current dependency for Jekyll is Ruby 2.1+. My inclination is to just specify that Jekyll depends on a certain version of ruby, and that a shell error will let you know if you don't have the right one. Otherwise the documentation risks becoming outdated if/when the Jekyll dependency changes (like if the next version of Jekyll requires 2.3+ then our docs would be wrong). What do you think? I'm not necessarily advocating for one thing over another, just putting the issue out there.

Reply:
I think a line as simple as:
"Install ruby and jekyll" could work. Maybe something about using RVM to manage ruby versions if you encounter issues running apps. I'm mostly concerned with one of our apps getting out of date and needing an old version of jekyll/ruby.